bin/prettier.js

```diff
@@ -9,25 +9,8 @@ const minimist = require("minimist");
 const prettier = require("../index");
 
 const argv = minimist(process.argv.slice(2), {
-  boolean: [
-    "write",
-    "stdin",
-    "single-quote",
-    "trailing-comma",
-    "bracket-spacing",
-    // The supports-color package (a sub sub dependency) looks directly at
-    // `process.argv` for `--no-color` and such-like options. The reason it is
-    // listed here is to avoid "Ignored unknown option: --no-color" warnings.
-    // See https://github.com/chalk/supports-color/#info for more information.
-    "color",
-    "help",
-    "version",
-    "debug-print-doc",
-    "debug-check",
-    // Deprecated in 0.0.10
-    "flow-parser"
-  ],
-  string: ["print-width", "tab-width", "parser"],
+  boolean: true,
```

Comment:
That is true about the breaking change, you need to type `--parser=flow`. Forcing an argument to `--trailing-comma` will be a breaking change too, and it's probably a more used option.

But maybe it's best to be explicit, and we should just require a parameter. I don't like the name "compat", can you think of a different name for that option?